src/store/LocalStorageStore.ts

```diff
@@ -16,11 +18,15 @@ export default class LocalStorageStore implements Store {
   }
 
   async get(key: string): Promise<Buffer | null> {
+    if (process.env.NODE_ENV === 'production') return null
```

Comment:
I think `process.env.NODE_ENV` will be `production` in the deployed version of our example chat (and certainly could be for other people's deployed apps. This will silently make the `LocalStorageStore` unusable in those cases.